snap-core/src/main/java/org/esa/snap/core/util/ProductUtils.java

```diff
@@ -657,6 +559,10 @@ private static void checkCanceled(ProgressMonitor pm) throws IOException {
         return pathList.toArray(new GeneralPath[pathList.size()]);
     }
 
+    /**
+     * @deprecated use {@link GeoUtils} instead
```

Comment:
Shouldn't the deprecated methods make use os the new GeoUtils methods?

Reply:
done